featurebyte/session/databricks.py

```diff
@@ -127,7 +127,7 @@ def upload_file_to_storage(
 
     def upload_dataframe_to_storage(self, dataframe: pd.DataFrame, remote_path: str) -> None:
         buffer = BytesIO()
-        dataframe.to_parquet(buffer)
+        dataframe.to_parquet(buffer, version="2.4")
```

Comment:
just forcing pyarrow to use a lower version when creating parquet files?

Reply:
yes the newest version is not yet properly supported by the latest version of Spark